.github/workflows/notify-fail.yml

```diff
@@ -0,0 +1,28 @@
+name: notify-fail
```

Comment:
Ah this is so cool. I didn't realise you could just put an independent job in ref the other jobs in workflows. Really nice to have it abstracted out to another file.